src/backend/OpenCL/CHIPBackendOpenCL.cc

```diff
@@ -743,9 +743,11 @@ void CHIPModuleOpenCL::compile(CHIPDevice *ChipDev) {
 
   std::string Log =
       Program.getBuildInfo<CL_PROGRAM_BUILD_LOG>(*ChipDevOcl->ClDevice, &Err);
-  if (ErrBuild != CL_SUCCESS)
+  if (ErrBuild != CL_SUCCESS) {
```

Comment:
Nit: unnecessary change noise and perhaps also non-conforming to the LLVM style.